articles/tutorials/building_2d_games/04_game_library/index.md

```diff
@@ -0,0 +1,126 @@
+---
+title: "Chapter 04: Game Library"
+description: Create a class library project to house reusable game components that we'll develop throughout this tutorial series.
+---
+
+Throughout this tutorial series, we will be creating reusable classes that add extended behavior on top of what MonoGame offers natively.  Instead of adding these classes directly to our game project, we can create a separate class library project to contain them.  This approach offers several advantages, including:
+
+1.  **Reusability**: The classes can be easily reused in future game projects by simply adding a reference to the library.
+2.  **Organization**: Keeps game-specific code separate from reusable library code.
+3.  **Maintainability**: Changes to the library can benefit all games that use it.
+4.  **Testing**: The library code can be tested independently of any specific game.
+
+A class library is a project type that compiles into a Dynamic Link Library (DLL) instead of an executable.  It contains reusable code that can be referenced by other projects, making it perfect for sharing common functionality across multiple games.  MonoGame offers the *MonoGame Game Library* project template that can be used to create a class library.
```

Comment:
Include a link to the Microsoft reference for what a DLL is :D (for new programmers)

Reply:
Link for MSDN article on DLLs included